tests/Phar/PharTest.php

```diff
@@ -0,0 +1,13 @@
+<?php declare(strict_types=1);
+
+namespace Rector\Tests\Phar;
+
+use PHPUnit\Framework\TestCase;
+
+final class PharTest extends TestCase
+{
+    public function test(): void
+    {
+        $this->assertTrue(file_exists(__DIR__ . '/../../rector.phar'));
```

Comment:
Maybe `assertFileExists`?

Reply:
This is WIP

Reply:
I'll follow your discussion with @theofidry and see what I can help :smile:

Reply:
Done :+1: